### ISSUE TITLE
Some videos from some channels

### ISSUE THREAD

**Issue by mrinter5** [ali1234/ytsubs#8]
I really needed a script like this for my project, so I decided to thoroughly test this one before actually committing to using it. I've opened my subscriptions box on youtube and compared all videos in there with the list produced by this script. What was strange is that one or two videos were always missing from the list. I've investigated further, and it seems like the playlistItems call lists videos from newest to oldest, but by the date of their upload, NOT the date of publishing. So, if a channel uploads a video, say, a month earlier than they publish it, and in the meantime publishes 5 or more videos, the aforementioned video will appear in the playlistItems list in a position that is higher than 5, leading to it not being included in the final output of this script.

My solution was to increase the limit in get_playlist_items function. Although still not 100% failproof, further testing revealed no more skipped videos. 

I am posting this not as an issue, but as information to whomever will be investigating this issue in the future.

**Comment by ali1234** [comment]
Yes, I've noticed this problem too. There's no way to work around it other
than to set the number of fetched videos excessively high, which will
increase your API usage.

On 13 February 2016 at 12:41, Algirdas <notifications@github.com> wrote:

> I really needed a script like this for my project, so I decided to
> thoroughly test this one before actually committing to using it. I've
> opened my subscriptions box on youtube and compared all videos in there
> with the list produced by this script. What was strange is that one or two
> videos were always missing from the list. I've investigated further, and it
> seems like the playlistItems call lists videos from newest to oldest, but
> by the date of their upload, NOT the date of publishing. So, if a channel
> uploads a video, say, a month earlier than they publish it, and in the
> meantime publishes 5 or more videos, the aforementioned video will appear
> in the playlistItems list in a position that is higher than 5, leading to
> it not being included in the final output of this script.
>
> My solution was to increase the limit in get_playlist_items function.
> Although still not 100% failproof, further testing revealed no more skipped
> videos.
>
> I am posting this not as an issue, but as information to whomever will be
> investigating this issue in the future.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ali1234/ytsubs/issues/8>.
>



-- 
Alistair Buxton
a.j.buxton@gmail.com
